### PR TITLE
#33 Updated index.html by correcting few more changes

### PR DESCRIPTION
**Pull request by shahkv95**
Additional solution to issue #33 

I have initially pulled a request number #34 which got merged in the master.
After that I found few more errors while reading the UI content. 
I have corrected it and making a PR for the same.
@madiedgar Please accept the PR and merge it.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -238,7 +238,7 @@ <h3 class="feature-title">Develop</h3>
                                     </div>
                                     <h3 class="feature-title">Learn</h3>
                                     <p class="text-sm">Let's face it - learning code is just as difficult as learning a new language.
-                                    </br></br>We're continually adding multilingual tutorials to our repository that includes introductions to programming, #OpenSource how-to’s and the Legesher fundamentals!</p>
+                                    </br></br>We're continually adding multilingual tutorials to our repository that includes introduction to programming, #OpenSource how-to’s and the Legesher fundamentals!</p>
```
Comment (madiedgar) [inline index.html]: Can you change the copy on this line to say something along the lines of "Stay tuned for multilingual tutorials for introduction to programming"? Would love to link our documentation website here, as that is the place that will host other tutorials and legesher documentation. 

Feel free to run your ideas by me!

File: index.html [hunk 2]
```diff
@@ -274,7 +274,7 @@ <h3 class="feature-title">Contribute</h3>
                                     </div>
                                     <h3 class="feature-title">Support</h3>
                                     <p class="text-sm">We're always wanting to add new spoken and programming languages to our repository. Please lege-<u>share</u> our mission.
-                                    </br></br>From beginners learning how to code to collaborating with diverse teams around the world, Legesher connects us all.</p>
+                                    </br></br>From beginners learning how to code to collaborate with diverse teams around the world, Legesher connects us all.</p>
```
Comment (madiedgar) [inline index.html]: To be honest, I'm not really crazy about this sentence in the first place. I've been thinking of another way to express the different use cases, but haven't found anything that really sticks. would love your thoughts!

File: index.html (hunk 1 above)
Comment (madiedgar) [inline index.html]: Can you change the copy on this line to say something along the lines of "Stay tuned for multilingual tutorials for introduction to programming"? Would love to link our documentation website here, as that is the place that will host other tutorials and legesher documentation.

Feel free to run your ideas by me!

File: index.html [hunk 3]
```diff
@@ -356,7 +356,7 @@ <h2 class="section-title mt-0">Stay in the know</h2>
                             </a>
                         </li>
                     </ul>
-                    <div class="footer-copyright">&copy; <a href="https://withmadi.co">  2018 With Madi Co.</a>, all rights reserved</div>
+                    <div class="footer-copyright">&copy; <a href="https://withmadi.co">  2018 With Madi Co.</a> | All rights reserved.</div>
```
Comment (madiedgar) [inline index.html]: Love this!

File: index.html (hunk 2 above)
Comment (shahkv95) [reply index.html]: Made the required changes @madiedgar

File: index.html (hunk 1 above)
Comment (shahkv95) [reply index.html]: Done @madiedgar

File: index.html (hunk 3 above)
Comment (shahkv95) [reply index.html]: Thank you @madiedgar

**Comment by shahkv95** [comment]
I have raised a PR for the changes. Please check it @madiedgar

**Pull request closed without merging**